aboveFold.css

```
@@ -20,7 +32,6 @@
     z-index: 1;
     text-align: center;
     max-width: 880px;
-    max-height: 382px;
     padding-top: 40px;
```

You can combine all paddings in one line:
```suggestion
    padding: 40px 0 40px 52px;
```